democracy/views/comment.py

```diff
@@ -54,7 +62,13 @@ def get_serializer_context(self):
 
     def get_queryset(self):
         queryset = super().get_queryset()
-        return queryset.filter(**{queryset.model.parent_field: self.get_comment_parent_id()})
+        filtered_queryset = queryset.filter(**{queryset.model.parent_field: self.get_comment_parent_id()})
+        request = self.get_serializer_context().get('request', None)
+        if request:
+            auth_code = request.GET.get('auth_code', None)
```

Comment:
The request arg should probably be called `authorization_code` too.

How about using some sort of actual DRF filtering mechanism instead of hard-coding only one filter though?